OpenTibia.Scheduling/BaseEvent.cs

```diff
@@ -132,7 +132,7 @@ public void Process()
 
                     sw.Stop();
 
-                    this.Logger.Verbose($"Executed ({i + 1} of {this.ActionsOnPass.Count}) on pass... done in {sw.Elapsed}.");
+                    // this.Logger.Verbose($"Executed ({i + 1} of {this.ActionsOnPass.Count}) on pass... done in {sw.Elapsed}.");
```

Comment:
let's re enable this, and bump some other logger events to Debug instead of picking what to comment out on the lowest.

(same below)

Reply:
done.